pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -375,6 +420,11 @@ stdenv.mkDerivation rec {
 
     wrapProgram "$out/bin/bazel" --add-flags --server_javabase="${runJdk}"
 
+    # generates the system bazelrc
+    # warning: the name of the repository depends on the system, hence
+    # the reference to .name
+    echo "build --override_repository=${remote_java_tools.name}=${remote_java_tools}" > $out/bazelrc
```

Comment:
Where does the system bazelrc end up?

Reply:
In `$out/bazelrc`. See earlier for the `BAZEL_SYSTEM_BAZELRC_PATH` replaced variable.

Reply:
I suggest moving the `bazelrc` into `$out/etc` or `$out/share`. It feels a bit odd to have it at the top-level of `$out`.

Also, aren't you missing `test` and `run`?